app/assets/javascripts/components/customer_search.jsx

```diff
@@ -20,44 +20,10 @@ var ListSearch = React.createClass({
     return (
       <div className="filter-list">
         <input type="text" placeholder="Search" onChange={this.filterList}/>
-      <Table items={this.state.items}/>
+        <Table items={this.state.items}/>
       </div>
     );
   }
 });
 
-var TableRow = React.createClass({
-  render: function(){
-    const{data} = this.props;
-    const row = data.map((data) =>
-      <tr className="row">
-        <td className="col s3 center" key={data.id}>{data.id}</td>
-        <td className="col s3 center" key={data.email}>{data.email}</td>
-      </tr>
-    );
-    return (
-      <span>{row}</span>
-    );
-  }
-})
-
-var Table = React.createClass({
-  render: function(){
-    console.log("logging this..", this.props)
-    return (
-      <table class="row s6">
-        <thead>
-          <tr className="row">
-            <td isKey={true} dataField="id" className="col s3 center">ID</td>
-            <td isKey={true} dataField="email" className="col s3 center">Email</td>
-          </tr>
-        </thead>
-        <tbody>
-          <TableRow data={this.props.items} />
-        </tbody>
-      </table>
-    );
-  }
-});
-
-React.render(<ListSearch/>, document.getElementById('mount-point'));
\ No newline at end of file
+React.render(<ListSearch/>, document.getElementById('mount-point'));
```

Comment:
Causing problems lol
I was screwing around with just React which is another way to render the component, but is not needed in this case.
I overlooked it